bes_theme/partials/logo.html

```diff
@@ -1,5 +1,5 @@
 {% if config.theme.logo %}
-  <img src="../assets/images/favicon.png" alt="logo">
+  <img src="./assets/images/favicon.png" alt="logo">
```

Comment:
I have used another method here to solve the issue #17 in this same file. So, it  might get a conflict. Can you please remove this.